core/config/engine.cpp

```diff
@@ -404,6 +409,19 @@ bool Engine::is_embedded_in_editor() const {
 	return embedded_in_editor;
 }
 
+void Engine::set_time_scale_base(double p_scale) {
+	time_scale_base = p_scale;
+}
+
```

Comment:
This function is duplicate to `set_time_scale`, and forgets to call `_update_time_scale()`, it should be removed.

```suggestion
```